packages/static-site/README.md

```diff
@@ -0,0 +1,12 @@
+# React + TypeScript + Vite
+
+- This app was created using the following code: `npm create vite@latest my-app -- --template react`
+
+- Similar to other quick start templates:
+`create-react-app`
+
+- Development server uses a custom host, which needs to be added to your machines host file `127.0.0.1 localhost.unity.web.asu.edu`
```

Comment:
Should be `127.0.0.1 localhost.asu.edu` to match what code expects and instructions on line 12.